core/payment/src/dao/invoice_event.rs

```diff
@@ -68,7 +68,7 @@ impl<'c> InvoiceEventDao<'c> {
                 .order_by(read_dsl::timestamp.asc())
                 .into_boxed();
             if let Some(timestamp) = after_timestamp {
-                query = query.filter(read_dsl::timestamp.gt(timestamp));
+                query = query.filter(read_dsl::timestamp.gt(timestamp.adapt()));
```

Comment:
[DISCUSSION_STARTER] I wonder is it possible to avoid these explicit fn calls and have it done somehow automatically?

Reply:
The problem is, that `NaiveDateTime` already implements `ToSql` trait, so I have to substitute new type here.

Other option would be to disable `chrono` feature in `diesel` which provides `ToSql` implementation. But I suspect we can't implement this trait ourselves, because both types are from external crates and rust doesn't allow this